packages/nodes-base/nodes/Google/Docs/GoogleDocs.node.ts

```diff
@@ -24,6 +24,9 @@ import {
 	IUpdateFields,
 } from './interfaces';
 
+import {
+	isEmpty
+} from 'lodash';
```

Comment:
No longer used?

Reply:
It makes the if statement more readable, but no need to use it .